.coveragerc

```diff
@@ -0,0 +1,6 @@
+[report]
+exclude_lines =
+    pragma: no cover
+    if __name__ == .__main__.:
+    _publish
+    main
```

Comment:
No linebreak, just a nitpicks if you care to fix https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline